R/pkg/inst/extdata/spark_download.csv

```diff
@@ -0,0 +1,2 @@
+"url","default"
+"http://apache.osuosl.org",TRUE
```

Comment:
Combined with the previous comment: It turns out this would be similar to what sparklyr did. They provide a remote table as well as one shipped with the package that lists supported versions and their download addresses.